### PR TITLE
tirePressureIndicators

### LINKED ISSUE #111: Change Tire Pressure Buttons to Increase/Decrease
- Have the Tire Pressure indicators measure the tire pressure
- Based off the tire pressure the indicator for low pressure should automatically show up when there is pressure lower than a given number 
- For test purposes, start your pressure at 35 PSI, and consider low pressure to be 30 PSI

### PR DESCRIPTION
**Pull request by Jyro88**
On changed to increase and Off to decrease. Pressure starts at 35 and low pressure considered 30. Pressing Off(decrease) 5 times will turn on the low pressure icon and pressing On after that should return back to normal

### CODE REVIEW

File: tirepressure.cpp [hunk 1]
```diff
@@ -19,10 +23,10 @@ void TirePressure::setup(Ui::MainWindow *ui, int height, int width) {
                  .scaled(carIconWH, carIconWH, Qt::KeepAspectRatio,
                          Qt::SmoothTransformation);
   this->ui->car_label->setPixmap(car_icon);
-  normal_pressure(FRONT_LEFT);
-  normal_pressure(FRONT_RIGHT);
-  normal_pressure(BACK_LEFT);
-  normal_pressure(BACK_RIGHT);
+  ui->front_left->setPixmap(normal_pressure_icon);
+  ui->front_right->setPixmap(normal_pressure_icon);
```
Comment (IseanB) [inline tirepressure.cpp]: you should check the tire pressure before you set it to normal.

File: tirepressure.cpp [hunk 2]
```diff
@@ -35,26 +64,51 @@ void TirePressure::setup(Ui::MainWindow *ui, int height, int width) {
   ui->car_label->move(screenWidth / 6 + tireIconWH * 1.8, screenHeight * .55);
 }
 
-void TirePressure::low_pressure(tires tire) {
+void TirePressure::increasePressure(tires tire) {
+
   if (tire == FRONT_LEFT) {
-    this->ui->front_left->setPixmap(low_pressure_icon);
+      frontLeftPressure++;
+      if(frontLeftPressure >= 31){
```
Comment (shahdivyank) [inline tirepressure.cpp]: make this a class variable `pressure_threshold`

**Review by shahdivyank — CHANGES_REQUESTED** [review]
What's up with the .gitignore file change?

File: tirepressure.cpp [hunk 3]
```diff
@@ -6,6 +6,42 @@ TirePressure::TirePressure() : lowpressure(34) { return; }
...
 
+  frontLeftPressure = 35;
+  frontRightPressure = 35;
+  backLeftPressure = 35;
+  backRightPressure = 35;
+  low_pressure_icon = QPixmap(":/icons/lowpressure.png");
+  normal_pressure_icon = QPixmap(":/icons/normalpressure.png");
+
+  
+  if(frontLeftPressure >= 31){
```
Comment (shahdivyank) [inline tirepressure.cpp]: make this a variable

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Review by IseanB — APPROVED** [review]

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**